ext/graphql_libgraphqlparser_ext/extconf.rb

```diff
@@ -1,6 +1,11 @@
 require 'mkmf'
 
-dir_config('graphql', '/usr/local/include/graphqlparser', '/usr/local/lib')
-abort 'missing libgraphqlparser' unless have_library 'graphqlparser'
+prefixes = %w(/usr /usr/local)
+prefix = prefixes.find{ |prefix| Dir["#{prefix}/lib/libgraphqlparser*"].first }
+abort 'missing libgraphqlparser' unless prefix
```

Comment:
Or, pick a prefix based on the OS? I think I picked `/usr/local` because that's where Homebrew put it